pipeline/live_pipeline.py

```diff
@@ -101,6 +103,7 @@ def handler(event: dict, context) -> None:
         for index, _ in enumerate(data):
             gpt_response.append(get_data(data, index))
         table_data = assemble_data(gpt_response)
+        send_emails(table_data, sns_client)
```

Comment:
Hi @MilesDrabwell , one thing to note, can we ensure that the 'log' variable is being passed into send_emails function, it will need to check if any story that satisfies the matched judge condition is also not already in the log, as otherwise repeat notifications will be sent out for previous stories in the same day.

Reply:
Looked into this and should be updated now.